doc/setup.md

```diff
@@ -141,41 +174,35 @@ output: {
 
 where:
 
-- `public_key[]` is an array of public keys generated by Signer[j] in Step 2.
-- `private_key[i]` is a private key of Signer[i] generated in Step 2.
+- `public_key[]` is an array of public keys generated by Signer[j].
+- `private_key[i]` is a private key of Signer[i] generated.
```

Comment:
> - `private_key[i]` is a private key of Signer[i] generated.

I think the word "generated" is not required.

```
`private_key[i]` is a private key of Signer[i].
```

Reply:
fixed.